kernel/syscall.c

```diff
@@ -3166,6 +3167,24 @@ long myst_syscall_get_process_thread_stack(void** stack, size_t* stack_size)
     return ret;
 }
 
+long myst_interrupt_thread(int tid)
```

Comment:
I'd suggest using different name (e.g., `myst_interrupt_host_thread`) or something else to distinguish with future support on interrupting thread while executing inside the enclave.

Reply:
Good idea!